samples/bookinfo/platform/kube/bookinfo-details-v2.yaml

```diff
@@ -29,7 +29,7 @@ spec:
     spec:
       containers:
       - name: details
-        image: istio/examples-bookinfo-details-v2:1.10.1
+        image: istio/examples-bookinfo-details-v2:1.11.1
```

Comment:
We've always updated the version of all bookinfo images together, even if only one of the services changed. That way it's much simpler to keep track of which version of bookinfo is being used.

I think we should publish the full set of 1.12.0 images next. Since @rvennam your code needs to do this too, I think we can rely on his PR to pull your new version of details too. https://github.com/istio/istio/pull/13314

Reply:
@frankbu @rvennam But his PR will break one test. This PR updates the version and updates a test. How about merging it and then merging Ram's PR?

Reply:
OK, that makes sense.

Reply:
@frankbu @rvennam I applied the comment of @frankbu and bumped all the versions to 1.13.0, in https://github.com/istio/istio/pull/13316/commits/9ab621b9779602bbab590a65d07ef544fb024fc4. Once release-1.1 is merged into master, we should bump the versions in master to 1.14.0, to accommodate all the changes in master and in release-1.1

Reply:
@vadimeisenbergibm I thought that the 1.12.0 images that @rvennam push to dockerhub would already include your change to the rating service, since the code change was already committed and he rebuilt the images before pushing. Is that not the case?

I also noticed that at the same time as you pushed the 1.13.0 images there were 5 other ratings images, with names like examples-bookinfo-ratings-v-xxx, also pushed (for example, https://cloud.docker.com/u/istio/repository/docker/istio/examples-bookinfo-ratings-v-buggy). What are those for?

Reply:
@frankbu The issue is that @rvennam merged his PR into master, while I merged my PR into release-1.1. This PR continues my PR into release-1.1. The 1.12.0 images do not contain my PR from release-1.1.

Regarding  examples-bookinfo-ratings-v-xxx, good catch! They are from my other work, the script pushed them automatically, I have removed them.